.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "lib/capistrano/tasks/trike"]
+	path = lib/capistrano/tasks/trike
+	url = git@git.trikeapps.com:trike/capistrano.git
+	branch = cap3_compatible_tasks
```

Comment:
I'm amazed this branch still exists. Would be good to know if we can point it at master - that branch hasn't been updated in _years_.